### PR TITLE
Activity Updates

### PR DESCRIPTION
**Pull request by jakekrog**
This branch makes the following activity changes:

Voice Activity
- Remove "Too Loud" indicators
- Update copy for instructions

Tapping Activity
- Remove old medication relevancy screen before active task

### CODE REVIEW

File: mPowerSDK/Resources/JSONs/Base.lproj/MedicationTracking.json [hunk 1]
```diff
@@ -106,10 +106,6 @@
         "optional"   : false,
      },
      {
-        "identifier" : "momentInDay",
-        "optional"   : false,
```
Comment (syoung-smallwisdom) [inline mPowerSDK/Resources/JSONs/Base.lproj/MedicationTracking.json:110]: NO!!! Do _not_ remove this. The desire is to ask both questions.

**Comment by syoung-smallwisdom** [comment]
DO NOT REMOVE "momentInDay" step. It is intentional to ask both questions as per @trister

File: mPowerSDK/TasksAndSteps/APHActivityManager.m [hunk 2]
```diff
@@ -172,8 +172,8 @@ - (ORKOrderedTask *)createCustomVoiceTask
     {
         ORKInstructionStep *instructionStep = (ORKInstructionStep *)orkTask.steps[instructionIdx];
         [instructionStep setTitle:localizedTaskName];
-        [instructionStep setText:NSLocalizedStringWithDefaultValue(@"APH_PHONATION_STEP_INSTRUCTION", nil, APHLocaleBundle(), @"Take a deep breath and say “Aaaaah” into the microphone for as long as you can. Keep a steady volume so the audio bars remain blue.", @"Instructions for performing the voice activity.")];
-        [instructionStep setDetailText:NSLocalizedStringWithDefaultValue(@"APH_NEXT_STEP_INSTRUCTION", nil, APHLocaleBundle(), @"Tap Next to begin the test.", @"Detail insctruction for how to begin a task.")];
+        [instructionStep setText:NSLocalizedStringWithDefaultValue(@"APH_PHONATION_STEP_INSTRUCTION", nil, APHLocaleBundle(), @"Hold your phone 6 inches from your mouth.", @"Instructions for performing the voice activity.")];
+        [instructionStep setDetailText:NSLocalizedStringWithDefaultValue(@"APH_NEXT_STEP_INSTRUCTION", nil, APHLocaleBundle(), @"Take a deep breath and say \"Aaaaaah\" into the microphone for as long as you can.", @"Detail insctruction for how to begin a task.")];
```
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m:176]: Cannot accept language changes without IRB approval.

File: mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m [hunk 3]
```diff
@@ -106,6 +108,27 @@ - (BOOL)taskViewController:(ORKTaskViewController *) __unused taskViewController
...
+}
+
+- (void)  taskViewController:(ORKTaskViewController *)taskViewController
+stepViewControllerWillAppear:(ORKStepViewController *)stepViewController {
+	if ([stepViewController.step.identifier isEqualToString:kInstructionStepIdentifier]) {
+		stepViewController.continueButtonTitle = NSLocalizedStringWithDefaultValue(@"BUTTON_GET_STARTED", @"ResearchKit", ORKBundle(),  @"Get Started", @"Get Started Button Text");;
+	} else if ([stepViewController.step.identifier isEqualToString:kInstruction1StepIdentifier]) {
+		stepViewController.continueButtonTitle = NSLocalizedStringWithDefaultValue(@"BUTTON_NEXT", @"ResearchKit", ORKBundle(),  @"Next", @"Next Button Text");;
+	}
+}
```
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m:130]: Cannot accept language changes without IRB approval.

File: mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m [hunk 4]
```diff
@@ -106,6 +108,27 @@ - (BOOL)taskViewController:(ORKTaskViewController *) __unused taskViewController
...
+						viewControllerForStep:(ORKStep *)step {
+	if ([step isKindOfClass:[ORKAudioStep class]]) {
+		ORKAudioStepViewController *stepViewController = [[ORKAudioStepViewController alloc] initWithStep:step];
+		//remove "too loud" red indicator; requires ResearchKit commit 42aa4408dc04b6f856cccf33fc3fb4484ac728c4
+		stepViewController.alertThreshold = 1.f;
+		return stepViewController;
+	}
+
+	return nil;
+}
```
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m:121]: Please submit a pull request with this code chunk to remove the red threshold indicator.

File: mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m [hunk 5]
```diff
@@ -239,6 +262,11 @@ - (BOOL)checkAudioLevelFromSoundFile:(NSURL *)fileURL
 {
     // Setup reader
     AVURLAsset * urlAsset = [AVURLAsset URLAssetWithURL:fileURL options:nil];
+    if (urlAsset.tracks.count == 0) {
+        NSLog(@"No tracks found for urlAsset: %@", fileURL);
+        return NO;
+    }
```
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m:268]: Please submit a pull request to add this bug fix.

File: mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m (hunk 4 above)
Comment (jakekrog) [inline mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m:121]: Opened separate PR: https://github.com/Sage-Bionetworks/mPowerSDK/pull/26

File: mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m (hunk 5 above)
Comment (jakekrog) [inline mPowerSDK/TasksAndSteps/PhonationControllers/APHPhonationTaskViewController.m:268]: Opened separate PR: https://github.com/Sage-Bionetworks/mPowerSDK/pull/27

**Comment by jakekrog** [comment]
Hi @syoung-smallwisdom,

I opened new PRs as you requested. I'll close this PR later tonight (unless there's something else you'd like merged in).

**Pull request closed without merging**